modules/bitgo/test/v2/unit/internal/tssUtils/eddsa.ts

```diff
@@ -221,17 +224,19 @@ describe('TSS Utils:', async function () {
       const nockedUserKeychain = await nockUserKeychain({ coin: coinName });
       await nockBackupKeychain({ coin: coinName });
 
-      const bitgoKeychain = await tssUtils.createBitgoKeychain(userGpgKey, userKeyShare, backupKeyShare);
-      const userKeychain = await tssUtils.createUserKeychain(
+      const bitgoKeychain = await tssUtils.createBitgoKeychain({ userGpgKey, userKeyShare, backupKeyShare: backupKeyShare });
```

Comment:
`backupKeyShare: backupKeyShare` seems to be unnecessary since the name matches the key name